SUI/SUI/Chair/ChairView.swift

```diff
@@ -0,0 +1,149 @@
+// ChairView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Экран со стулом
```

Comment:
```suggestion
/// Экран со стулом
```
Странное описание) кажется Женя его может завернуть)
можно что-то типо Описание товара или Экран товара